prime-angular-frontend/src/app/core/services/utils.service.ts

```diff
@@ -133,4 +133,27 @@ export class UtilsService {
   public downloadToken(token: string): void {
     window.location.assign(`${this.config.documentManagerUrl}/documents/downloads/${token}`);
   }
+
+  /**
+   * @description
+   * Opens a mailto href from your current window
```

Comment:
The comment could be a bit more descriptive like: "Open an email using the user's default mail client."